src/extensions/registries/__tests__/page-registry.test.ts

```diff
@@ -17,6 +17,16 @@ describe("getPageUrl", () => {
       isBundled: false,
       isEnabled: true
     });
+    globalPageRegistry.add({
+      id: "page-with-params",
+      components: {
+        Page: () => React.createElement("Page with params")
+      },
+      params: {
+        test1: "test1-default",
+        test2: "" // no default value, just declaration
```

Comment:
I don't think that this should be required. If it isn't all good.

Reply:
It should be required since it's declaration. What kind of params page willing to accept therefor use in any-way (e.g. in menus with default values).